.github/actions/lighthouse-performance-testing/action.yml

```diff
@@ -19,9 +19,53 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Install dependencies
```

Comment:
@Thebarda You might want to use: https://github.com/shivammathur/setup-php

Reply:
wow